ui/page/settings/app_settings_page.go

```diff
@@ -267,6 +287,18 @@ func (pg *AppSettingsPage) general() layout.Widget {
 				layout.Rigid(func(gtx C) D {
 					return pg.subSectionSwitch(gtx, values.String(values.StrTxNotification), pg.transactionNotification)
 				}),
+				layout.Rigid(func(gtx C) D {
+					if !pg.AssetsManager.DEXCInitialized() || len(pg.AssetsManager.DexClient().Exchanges()) == 0 {
+						return D{}
+					}
```

Comment:
Allow backup even if exchanges have not been added yet. As long as dexc has been initialized with a password.